modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/DataApplicationSnapshotAcceptanceManager.scala

```diff
@@ -181,13 +180,14 @@ object DataApplicationSnapshotAcceptanceManager {
           validatedBlocks.traverse(service.serializeBlock)
         )
 
+        feeTransactions <- OptionT.liftF {
+          getFeeTransactions(validatedUpdates).pure
+        }
```

Comment:
isn't it: `feeTransactions = getFeeTransactions(validatedUpdates)` or sth equivalent? You don't need to lift a pure value 🤔

Reply:
👍🏼